### ISSUE TITLE
Enquiry about Baseline result using resnet50

### ISSUE THREAD

**Issue by petertran1811** [nixingyang/FlipReID#8]
Dear @nixingyang 

I would like to ask about the results that you proposed in your paper when using resnet50 as backbone, which are mAP@61.7  and R1@81.5 in MSMT dataset. I looked back at the paper that you adapt the code "Adaptive L2 Regularization in Person Re-Identification" and their results for resnet50 was just 57.7% and 79.1% for mAP and R1 respectively. I could not figure out why do you have such a good improvement; therefore, I really look forward to hearing your opinions about it. Hopefully, I do not cause any convenience for you. 

Thank you in advance!

**Comment by nixingyang** [comment]
Hi,
Thanks for your interest in our work.
In [AdaptiveL2Regularization](https://arxiv.org/abs/2007.07875), we used the [official TensorFlow/Keras backbones](https://github.com/nixingyang/AdaptiveL2Regularization/blob/master/applications). Based on this [table](https://github.com/keras-team/keras-applications), the top-1 accuracy of `ResNet50` is 74.928.
In [FlipReID](https://arxiv.org/abs/2105.05639), we used the backbones provided in [imgclsmob](https://github.com/nixingyang/FlipReID/blob/master/applications). Based on this [table](https://github.com/osmr/imgclsmob/tree/master/tensorflow2), the top-1 accuracy of `ResNet-50` is 100-22.09==77.91.
As far as I can remember, the main reason for the performance boost is the change of the pre-trained backbones. I might have also made some minor modifications here or there.
All the best.
Xingyang

**Comment by petertran1811** [comment]
Thank you. I got it.
